Client/game_sa/CStreamingSA.cpp

```diff
@@ -51,13 +49,179 @@ namespace
             memoryUsed = *(DWORD*)VAR_CStreaming_memoryUsed;
         }
     };
-}            // namespace
+
+    constexpr size_t MAX_IMAGES_NUM = 0xFF;
+    constexpr size_t MAX_STREAMS_NUM = 0xFF;
```

Comment:
I remember steam handlers used for game audio. So MAX_IMAGES_NUM can't be 255

Reply:
It is quite possible, but I cannot find an evidence of this. If you could indicate the location of it, I would appreciate it.

Reply:
GTA can't handle more that 246 IMG archives.
By default it uses 16 stream handlers and 6 IMG archives. 10 stream handlers used for audio. So 256 - 10 = 246
So MAX_IMAGES_NUM should be 255, it there isn't any other limitations.

Reply:
I have done a bit of research and can tell you that you're right. Perhaps with the exception of the amount of audio streams(it's 9 streams). Actually SA used 10 reserved handles(9 for audio + 1 for GTA3.IMG(yes, there are two GTA3's handles). The full list of handles in the default game: "MODELS\\GTA3.IMG", "AUDIO\\SFX\\FEET", "AUDIO\\SFX\\GENRL", "AUDIO\\SFX\\PAIN_A", "AUDIO\\SFX\\SCRIPT", "AUDIO\\SFX\\SPC_EA", "AUDIO\\SFX\\SPC_FA", "AUDIO\\SFX\\SPC_GA", "AUDIO\\SFX\\SPC_NA", "AUDIO\\SFX\\SPC_PA", "MODELS\\GTA3.IMG", "MODELS\\GTA_INT.IMG", "DATA\\PATHS\\CARREC.IMG ", "DATA\\SCRIPT\\SCRIPT.IMG ", "MODELS\\CUTSCENE.IMG ",  "MODELS\\PLAYER.IMG". I've taken all the points in the last commit.